pubsub/rabbitmq/metadata.go

```diff
@@ -46,6 +47,8 @@ type metadata struct {
 	maxLen           int64
 	maxLenBytes      int64
 	exchangeKind     string
+	queueType        string
```

Comment:
While allowing Stream queue types as well, it seems from documentation that it may be best practice to set max-age and max-length-bytes for it (https://rabbitmq.com/streams.html#retention):
```
Streams are implemented as an immutable append-only disk log. This means that the log will grow indefinitely until the disk runs out. To avoid this undesirable scenario it is possible to set a retention configuration per stream which will discard the oldest data in the log based on total log data size and/or age.
```
max-length-bytes already seems to be supported here, Should we add at least `max-age` also here (if not others like x-stream-max-segment-size-bytes)?

Reply:
Sure. I will add in the max-age arguments.